src/SmartComponents/AdvisoryDetail/CveModal.test.js

```diff
@@ -29,107 +22,80 @@ jest.mock('../../Utilities/api', () => ({
 const mockState = { ...storeListDefaults, rows: cveRows };
 
 const initStore = (state) => {
-    const customMiddleWare = () => next => action => {
-        useSelector.mockImplementation(callback => {
-            return callback({ CvesListStore: state });
-        });
-        next(action);
-    };
-
-    const mockStore = configureStore([customMiddleWare]);
+    const mockStore = configureStore([]);
     return mockStore({ CvesListStore: state });
 };
 
-let wrapper;
 let store = initStore(mockState);
 
 beforeEach(() => {
-    store.clearActions();
-    useSelector.mockImplementation(callback => {
-        return callback({ CvesListStore: mockState });
-    });
     createCvesRows.mockImplementation(() => readyCveRows);
 
-    wrapper = mount(<Provider store={store}>
-        <Router><CvesModal /></Router>
-    </Provider>);
-});
-
-afterEach(() => {
-    useSelector.mockClear();
+    render(<ComponentWithContext renderOptions={{ store }}>
+        <CvesModal />
+    </ComponentWithContext>);
 });
 
+//TODO: convert disabled tests to RTL after react&paterrnfly migration
```

Comment:
What are the issues with converting them before?

Reply:
This modal does not get used in patch according to the pendo. I wanted to unblock the migrations for Patch by converting essential tests in the first place. I will start working on this and deliver it as soon as possible